### PR TITLE
Abnormal security C2C doc update

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 1]
```diff
@@ -77,6 +80,7 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
 | category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"`
 | fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
 | accessToken | String | Yes | `null`| Access token used to retrieve abnormal threats. |  |
+| datacollection | array | Yes | `Threat`| List of APIs to fetch the data from. | threat, cases |
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:83]:
```suggestion
| datacollection | array | Yes | `Threat`| List of APIs to fetch the data from. | `threat, cases` |
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ sidebar_label: Abnormal Security
 tags:
   - cloud-to-cloud
   - abnormal-security
-description: Learn how to collect abnormal threat logs from the Abnormal Security source.
+description: Learn how to collect abnormal threat and cases logs from the Abnormal Security source.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:8]:
```suggestion
description: Learn how to collect abnormal threat and case logs from the Abnormal Security source.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Abnormal Security is a behavioral AI-based email security platform that learns the behavior of every identity in a cloud email environment and analyzes the risk of every event to block the most sophisticated attacks.
 
-The Abnormal Security integration ingests threat data identified by the abnormal threat log using the [Abnormal Security API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1).
+The Abnormal Security integration ingests threat data and cases data identified by the abnormal threat log and cases using the [Abnormal Security API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:20]:
```suggestion
The Abnormal Security integration ingests threat data and case data identified by the abnormal threat log and cases using the [Abnormal Security API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 4]
```diff
@@ -77,6 +79,7 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
 | category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"`
 | fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
 | accessToken | String | Yes | `null`| Access token used to retrieve abnormal threats. |  |
+| datacollection | array | Yes | `Threat`| List of APIs to fetch the data from. | `threat, cases` |
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:82]:
```suggestion
| datacollection | array | Yes | `threats`| List of APIs to fetch the data from. | `threats, cases` |
```

**Comment by jpipkin1** [comment]
Merging at the request of @yleiferman.

**Pull request merged**